packages/backend/src/utils/podman.ts

```diff
@@ -16,7 +16,10 @@
  * SPDX-License-Identifier: Apache-2.0
  ***********************************************************************/
 import type { ProviderContainerConnection } from '@podman-desktop/api';
-import { configuration, env, process, provider } from '@podman-desktop/api';
+import { configuration, containerEngine, env, navigation, process, provider } from '@podman-desktop/api';
+import type { ContainerConnectionInfo } from '@shared/src/models/IContainerConnectionInfo';
+
+export const MIN_CPUS_VALUE = 10;
```

Comment:
Wondering if this is not quite a high value

Reply:
I just copied what it was on the mockup. Which value should i set? 4-6-8?

Reply:
I would choose 4 but let me give a try

Reply:
Edited with 4 for the moment, gonna give it a try on the mac to see if it runs fine